wakka_auth/wakka/views.py

```diff
@@ -256,22 +241,14 @@ def get(self, request: Request, *args, **kwargs):
             return render(
                 request,
                 "forgot_password_page.html",
-                {
```

Comment:
Make for forgot password form too, as follows.
{
    "title": FORGOT_PASSWORD_FORM_TITLE,
    "description": None,
    "type": "FORM",
    "token": reset_password_form_token,
},

into 
{
    **FORGOT_PASSWORD_FORM_PAGE_CONTEXT,
    "token": reset_password_form_token
}